src/main/java/org/checkerframework/specimin/UnsolvedSymbolVisitor.java

```diff
@@ -182,8 +193,8 @@ private void setclassAndPackageMap() {
    *
    * @return superClass the value of superClass
    */
-  public String getSuperClass() {
-    return superClass;
+  public Collection<@ClassGetSimpleName String> getSuperClass() {
+    return classAndItsParent.values();
```

Comment:
It is surprising that a method called `getSuperClass` returns a collection. The documentation should explain what the meaning of the returned collection is, and how to actually get the name of the super class.